vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -367,7 +367,10 @@ func (feeder *clusterStateFeeder) LoadPods() {
 		}
 		feeder.clusterState.AddOrUpdatePod(pod.ID, pod.PodLabels, pod.Phase)
 		for _, container := range pod.Containers {
-			feeder.clusterState.AddOrUpdateContainer(container.ID, container.Request)
+			err = feeder.clusterState.AddOrUpdateContainer(container.ID, container.Request)
```

Comment:
nit: You can shorten this into if err=..; err != nil (applies in the below changes as well)